### PR TITLE
SDI-355 Make sure filters always have time element

### PR DESCRIPTION
**Pull request by andymarke**

### CODE REVIEW

File: server/routes/incentiveMigration/incentivesMigrationController.ts [hunk 1]
```diff
@@ -34,6 +35,8 @@ export default class IncentivesMigrationController {
     } else {
       const { migrations } = await this.nomisMigrationService.getIncentivesMigrations(context(res), {
         ...searchFilter,
+        toDateTime: withDefaultTime(searchFilter.toDateTime),
+        fromDateTime: withDefaultTime(searchFilter.fromDateTime),
       })
```
Comment (andymarke) [inline server/routes/incentiveMigration/incentivesMigrationController.ts:40]: without this users are forced to add time element that is a pain, was already present for visits but missed it from incentives

File: server/routes/visitMigration/visitsMigrationController.ts [hunk 2]
```diff
@@ -174,13 +175,6 @@ export default class VisitsMigrationController {
     return value
   }
 
-  private static withDefaultTime(value?: string): string | undefined {
```
Comment (andymarke) [inline server/routes/visitMigration/visitsMigrationController.ts:177]: moved to utils

**Review by petergphillips — APPROVED** [review]
LGTM

File: server/utils/utils.ts [hunk 3]
```diff
@@ -14,4 +16,11 @@ const properCaseName = (name: string): string => (isBlank(name) ? '' : name.spli
 const convertToTitleCase = (sentence: string): string =>
   isBlank(sentence) ? '' : sentence.split(' ').map(properCaseName).join(' ')
 
+export const withDefaultTime = (value?: string): string | undefined => {
+  if (value) {
+    return moment(value).format('YYYY-MM-DDTHH:mm:ss')
```
Comment (petergphillips) [inline server/utils/utils.ts:21]: I think we're trying to move away from moment, but this is an internal service anyway so fine.

**Pull request merged**